docs/developer-guide/shipping-bytecode.md

```diff
@@ -36,35 +36,34 @@ a `.o` extension) placed at the root of the layer `./`.
 To provide relevant metadata regarding the bytecode to any consumers, some relevant labels
 **MUST** be defined on the image.
 
-These labels are defined as follows:
+These labels are dynamic and defined as follows:
 
-- `io.ebpf.program_type`: The eBPF program type (i.e `xdp`,`tc`, `sockops`, ...).
+- `io.ebpf.programs`: A label which defines The eBPF programs stored in the bytecode image.
```

Comment:
Is it possible (as a future enhancement) to automatically pull the lists of programs and maps from the bytecode?

Reply:
Strictly speaking, I don't think they are required for proper operation as we've demonstrated with the multi-program BpfApplication images.  I think our code just uses them as a sanity check when loading.  If we didn't do this check, I think the Aya code would barf if the user tried to load something that didn't exist in the image.  However, I think they are useful for documenting what is contained in the image.

Reply:
No enhancement needed `bpfman image build` already does this (added in this PR)!! 🥳

Reply:
I guess I missed that.